tests/lax_numpy_test.py

```diff
@@ -1992,6 +1992,23 @@ def testRavel(self):
     args_maker = lambda: [rng.randn(3, 4).astype("float32")]
     self._CompileAndCheck(lambda x: x.ravel(), args_maker, check_dtypes=True)
 
+  @parameterized.parameters(
+    (0, (2, 1, 3)),
+    (5, (2, 1, 3)),
+    (0, ()))
```

Comment:
numpy's version also supports array inputs for the first entry; e.g.
```
>>> np.unravel_index([22, 41, 37], (7,6))
(array([3, 6, 6]), array([4, 5, 1]))
```
It would be useful to test this case as well.